src/document-click.js

```diff
@@ -108,6 +113,20 @@ export function onDocumentElementClick_(e, viewport) {
   const win = doc.defaultView;
 
   const tgtLoc = parseUrl(target.href);
+
+  // On Safari iOS, custom protocol links will fail to open apps when the
+  // document is iframed - in order to go around this, we set the top.location
+  // to the custom protocol href.
+  const isSafariIOS = platform.isIos() && platform.isSafari();
+  const isEmbedded = win.parent && win.parent != win;
+  const isNormalProtocol = /^https?:$/.test(tgtLoc.protocol);
+  if (isSafariIOS && isEmbedded && !isNormalProtocol) {
+    win.open(target.href, '_blank');
```

Comment:
Certainly, that's understood. But it does mean that going with `blank_` is sufficient right now. if we want to change behavior to `top_` - we'd want to do it cdn to start with.